api/types/accesslist/member.go

```diff
@@ -90,6 +90,9 @@ func (a *AccessListMember) CheckAndSetDefaults() error {
 		return trace.Wrap(err)
 	}
 
+	if a.Spec.Name == "" {
```

Comment:
nit: I would recommend add a comment explaining this behavior and why this is safe.